Python/Product/Analysis/Intellisense/ProjectEntryMap.cs

```diff
@@ -47,13 +48,13 @@ sealed class ProjectEntryMap : IEnumerable<KeyValuePair<string, IProjectEntry>>
                 // ids are 1 based
                 id = _ids.Count;
             }
-            _projectFiles[filename] = node;
+            _projectFiles[new Uri(filename, UriKind.RelativeOrAbsolute)] = node;
```

Comment:
Is it ok to have both relative and absolute paths in the same map?

Reply:
This entire class is unused now - I'll delete it.